diff --git a/scripts/cmake/doctestAddTests.cmake b/scripts/cmake/doctestAddTests.cmake
index e559893..57bd990 100644
--- a/scripts/cmake/doctestAddTests.cmake
+++ b/scripts/cmake/doctestAddTests.cmake
@@ -36,6 +36,7 @@ if("${spec}" MATCHES .)
 endif()
 
 execute_process(
+  ENCODING UTF-8
   COMMAND ${TEST_EXECUTOR} "${TEST_EXECUTABLE}" ${spec} --list-test-cases
   OUTPUT_VARIABLE output
   RESULT_VARIABLE result
@@ -61,6 +62,7 @@ foreach(line ${output})
   if(${add_labels})
     # get test suite that test belongs to
     execute_process(
+      ENCODING UTF-8
       COMMAND ${TEST_EXECUTOR} "${TEST_EXECUTABLE}" --test-case=${test} --list-test-suites
       OUTPUT_VARIABLE labeloutput
       RESULT_VARIABLE labelresult
